index.html

```diff
@@ -6,27 +6,98 @@
    </head>
    <body>
       <section class="section">
-          <h1 class="title">Algorithms time Benchmarking</h1>
-          <h2 class="subtitle">
-             Audio comparison between  <strong>Meydas</strong> and <strong>Essentia</strong>
-          </h2>
-         <div class="container">
+		<h1 class="title">Algorithms time Benchmarking</h1>
+		<h2 class="subtitle">
+			Audio comparison between  <strong>Meyda.js</strong> and <strong>essentia.js</strong>
+		</h2>
+		<div class="container" id="rms">
             <div class="notification">
-               <h1 class="title">RMS</h1>
-               <div class="columns is-mobile is-centered">
-                  <div class="column is-half">
-                     <div class="buttons">
-                        <button id="rms_offline" class="button is-primary">Offline</button>
-                     </div>
-                  </div>
-                  <div class="column">
-                     <div class="buttons">
-                        <button class="button is-primary">Online</button>
-                     </div>
-                  </div>
-               </div>
-               <div>
-                  <p id="results_rms"></p>
+				<h1 class="title">RMS</h1>
+				<div class="columns is-mobile is-centered">
+					<div class="column">
+						<div class="buttons">
+						    <button id="rms_offline" class="button is-primary">Offline</button>
+						</div>
+						<p id="results_rms"></p>
+						<a id="download_results_rms" class="is-hidden"></a>
```

Comment:
I would construct all the childs of results by javascript.. but it would be another task.

Reply:
M'has picat. On it! XD